js/templates/pageNav.html

```diff
@@ -8,7 +8,7 @@ <h3 class="margin0 marginTop25"><%= polyglot.t('Discover') %></h3>
   </a>
 </div>
 
-<div class="popMenu hide popMenu-navBar js-navProfileMenu">
+<div class="popMenu popMenu-navBar js-navProfileMenu">
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1584275/13321893/c0e8349c-db9f-11e5-8bd0-3300b74535d3.png)

On Windows this is still visible. Changing the top property to -300px fixes it.